src/Welcome/main.py

```diff
@@ -19,7 +21,7 @@ def dialog_response(dialog, response):
     dialog.close()
 
 
-subtitle_box: Gtk.Box = workbench.builder.get_object("subtitle")
+subtitle_box = cast(Gtk.Box, workbench.builder.get_object("subtitle"))
```

Comment:
This is needed, otherwise mypy complains:

```
src/Welcome/main.py:22: error: Incompatible types in assignment (expression has type "Object | None", variable has type "Box")  [assignment]
Found 1 error in 1 file (checked 1 source file)
```

We will need to add these casts everywhere where the concrete return type can vary.

Reply:
`cast` is just the identity function at runtime: https://docs.python.org/3/library/typing.html#typing.cast